src/ADAL.PCL.iOS/TokenCachePlugin.cs

```diff
@@ -26,12 +29,66 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
     internal class TokenCachePlugin : ITokenCachePlugin
     {
+        private const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```

Comment:
iOS uses MSOpenTech.ADAL.1 as the generic name. your container should be named the same otherwise if the developer moves to native library, and updates the app, the keychain data will not be accessible.